DESIGN.md

```diff
@@ -72,49 +75,49 @@ to three strong requirements:
 
 ## Existing designs?
 
-There are of course, many different existing filesystem. Heres a very rough
+There are of course, many different existing filesystem. Here is a very rough
 summary of the general ideas behind some of them.
 
 Most of the existing filesystems fall into the one big category of filesystem
-designed in the early days of spinny magnet disks. While there is a vast amount
-of interesting technology and ideas in this area, the nature of spinny magnet
+designed in the early days of spinning magnetic disks. While there is a vast amount
+of interesting technology and ideas in this area, the nature of spinning magnetic
```

Comment:
How strongly do you feel about "spinning magnetic disks". I prefer the tongue in cheek "spinny magnet disks", but if you think it's inappropriate I'm fine changing it.

Reply:
I thought it might have been sloppyness when typing but if it was chosen consciously then sure, let's keep what you had. Reverted.